src/AddTodoForm.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+const AddTodoForm = () => {
+  return (
+    <form>
+      <label htmlFor="todoTitle">Title</label>
+      <input type="text" id="todoTitle" />
```

Comment:
htmlFor semantically connects this label to its associated input field which is really important for users who use assistive technologies to use websites. 👍